### DISCUSSION TITLE
Considering alternative chat services/bridges for development communication

### DISCUSSION THREAD

**Discussion by gabeklavans** [input-leap/input-leap#1785] [Development of Input Leap]
I like using GitHub discussions for topic-based conversation, and IRC is great for simple ad-hoc communication, but I believe that using solely IRC for general development discussion adds a bit of a barrier to entry. I personally don't have any persistent IRC solution set up, and it takes a bit of know-how to do so, so I may miss messages in the chat or people may miss mine. Maybe I'm just naive here, but I think considering adding something like a Telegram/Matrix group to the project (or some sort of bridging solution) could help newcomers onboard quicker and add more developers to the pot.

**Comment by gabeklavans** [comment]
Or maybe, the "barrier to entry" I mentioned could be considered a feature, not a bug :stuck_out_tongue: either way just thought I'd start the discussion on this topic.

**Comment by ann0see** [comment]
I believe a Matrix, IRC maybe Discord bridge would lower the entry largely. But if it's too open this would mean more maintenance work.

**Comment by shymega** [comment]
I've configured an IRC <-> Matrix bridge tonight - feel free to try it [out](https://matrix.to/#/#input-leap:matrix.org)!

I'm gonna try and get anti-spam bots sorted, then I'll bridge to a Discord server.

Once I'm happy with the setup, I'll make an announcement.

**Reply by shymega** [reply]
Discord is now setup.
